### PR TITLE
Adding a basic query widget to items html

### PR DESCRIPTION
**Pull request by pvgenuchten**
Initial implementation for #451

image

Current limitations:
- does not use the queryables as advertised in /collections/{col}/queryables
- no option to see/cancel the current filters
- only support field=value for now, when CQL is available we can extend it to support other operators such as like, >, <, etc.

Is it correct that filters are not operational on the geojson-backend ?

### CODE REVIEW

**Comment by pvgenuchten** [comment]
Any feedback here? people like the approach as a start, any suggestions for improvement?

**Comment by tomkralidis** [comment]
cc'ing @kngai for any UI/UX comments.

Nice work @pvgenuchten . Correct that the native GeoJSON driver doesn't support filters.

We'll need some sort of advertisement from providers on what they support filter wise (see #403)

File: pygeoapi/templates/items.html [hunk 1]
```diff
@@ -16,12 +16,30 @@
...
+            {% endfor %}
+          </select>
+          <select style="height:40px;width:60px">
+            <option>=</option>
+          </select>
+          <input type="text" id="filterfield" value="" />
+          <input type="submit" value="Filter"/>
+        </FORM>
+      </div>
+    </div>
```
Comment (kngai) [inline pygeoapi/templates/items.html:41]: Excellent stuff here. My only minor suggestions are:
- Place this `<div class="row">` block under line 43 (`{% if data['features'] %}`) so it renders with the table
- Change `<FORM>` to `<form>`

**Comment by pvgenuchten** [comment]
updated the pr to populate the filters from queryables
@kngai I'd suggest to move/improve visuals after merge

**Comment by pvgenuchten** [comment]
replaced by #609

**Pull request closed without merging**